src/TerminalFighter/main.cpp

```diff
@@ -135,9 +138,15 @@ int main(int argc, char* argv[]) {
     google::InitGoogleLogging(argv[0]);
     LOG(INFO) << "Logging Intialized INFO";
 
-    bool high_dpi = settings.video_settings()["high_dpi"].as<bool>();
-    int screen_width = settings.video_settings()["window"]["width"].as<int>();
-    int screen_height = settings.video_settings()["window"]["height"].as<int>();
+    bool high_dpi = false;
+    int screen_width = 0;
+    int screen_height = 0;
```

Comment:
Just curious, is there a point to having these start with a default value? If the settings don't load, we're just terminating the program so these don't actually have to start off with any value right?

Reply:
Better to just do it safely? Good habit to initialize variables.